napari/layers/points/points.py

```diff
@@ -296,25 +305,20 @@ def __init__(
         self._data = np.asarray(data)
 
         # Save the properties
-        if properties is None:
+        if self.data.size == 0 and properties:
+            warnings.warn(
+                trans._(
+                    "Property_choices should be passed as property_choices, not properties. "
+                    "Workaround will be removed in 4.11"
+                ),
```

Comment:
```suggestion
                    "Property choices should be passed as property_choices, not properties. This warning will become an error in version 0.4.11.",
                ),
```

Translated strings need to be on one line until we improve our translation engine.